src/main/java/com/textadventure/Lexer.java

```diff
@@ -7,8 +7,7 @@
 public class Lexer {
 
 	public enum TokenType {
-		MOVEMENT(" w | a | s | d "), WALK("walk"), RUN("run"), DIRECTION("North|South|East|West"),
-		WHITESPACE("[ \t\f\r\n]+");
+		WALK("walk"), RUN("run"), DIRECTION("North|South|East|West");
 		public final String pattern;
 
 		private TokenType(String pattern) {
```

Comment:
Redundant **private modifier** as constructor without a modifier is already a private constructor.